### DISCUSSION TITLE
Expecting blockscout 2.0 as our new Blockchain explorer

### DISCUSSION THREAD

**Discussion by nnlgsakib** [blockscout/blockscout#7783] [Q&A] [answered]
Hello. I am Sakib. I hope you are well.

I am employed as the Chief Developer of a Blockchain  called MIND Smart Chain.

As an explorer for our Blockchain we are interested in getting access to Blockscout 2.0

Telegram:- @SakibNL1

**Comment by sl1depengwyn** [comment, marked as answer]
There is a discussion for this https://github.com/orgs/blockscout/discussions/7121

**Comment by Shednandezstore** [comment]
Depends on the size of the page
